lib/widgets/app.dart

```diff
@@ -117,7 +118,12 @@ class _ZulipAppState extends State<ZulipApp> with WidgetsBindingObserver {
         final initialAccountId = globalStore.accounts.firstOrNull?.id;
         return MaterialApp(
           title: 'Zulip',
-          localizationsDelegates: ZulipLocalizations.localizationsDelegates,
+          localizationsDelegates: const [
+            ...ZulipLocalizations.localizationsDelegates,
+            GlobalMaterialLocalizations.delegate,
+            GlobalWidgetsLocalizations.delegate,
+            GlobalCupertinoLocalizations.delegate,
```

Comment:
nit: place in logical order:
```suggestion
            ...ZulipLocalizations.localizationsDelegates,
            GlobalMaterialLocalizations.delegate,
            GlobalCupertinoLocalizations.delegate,
            GlobalWidgetsLocalizations.delegate,
```

If our app-specific delegates go first, then the next-most-specific should go next, and so on to the most general delegate going at the end. The `flutter/material` package depends on `flutter/cupertino` which depends on `flutter/widgets`, so that gives the ordering.